fri/src/two_adic_pcs.rs

```diff
@@ -272,8 +273,11 @@ where
                 rounds
                     .iter()
                     .map(|(data, _)| {
-                        // needs to recombine decomposed openings.. or something...
```

Comment:
ugh, I need to be more explicit in my TODOs - I think this was referring to the unused API I had where it would automatically decompose polys with too high degree. so not relevant